{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -360,6 +360,28 @@ def outgoing_commits(c):
     print_commits(get_outgoing_commits(c))
 
 
+@task
```

Comment:
Currently, it does not make sense as a task because it does not output anything to the console